scripts/Create-Win32App.ps1

```diff
@@ -61,7 +61,7 @@ process {
         $AppIconFile = $OutFile
     }
     else {
-        $AppIconFile = $AppData.PackageInformation.IconFile
+        $AppIconFile = [System.IO.Path]::Combine((Get-Item $Json).DirectoryName, $AppData.PackageInformation.IconFile)
```

Comment:
I believe there is a proposal to accept `$AppPath` as an argument to this script without having to do this